### PR TITLE
Add test for `TagFoodCommand` and `TagFoodCommandParser`

### PR DESCRIPTION
**Pull request by Jnjy**
More test to be added but due to some changes made may be needed for testing of other class.

Useful classes created for testing are:
- `TypicalTag.java`
- `TypicalWife.java`

### CODE REVIEW

File: src/test/java/seedu/wife/testutil/TypicalFood.java [hunk 1]
```diff
@@ -3,46 +3,57 @@
...
-    }
-
     public static List<Food> getTypicalFood() {
         return new ArrayList<>(Arrays.asList(MEIJI, CHOCOLATE));
     }
+
+    public static List<Food> getTypicalFoodWithoutTag() {
+        return new ArrayList<>(Arrays.asList(MEIJI_WITHOUT_TAG, CHOCOLATE_WITHOUT_TAG));
+    }
 }
```
Comment (ScorpiusSigma) [inline src/test/java/seedu/wife/testutil/TypicalFood.java:59]: Can we add a TypicalFood with both tags and no tags?

File: src/test/java/seedu/wife/testutil/TypicalTag.java [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+ */
+public class TypicalTag {
+    public static final Tag DAIRY_TAG = new TagBuilder()
+            .withTagName(VALID_TAG_DAIRY)
+            .build();
+
+    public static final Tag CHOCOLATE_TAG = new TagBuilder()
+            .withTagName(VALID_TAG_CHOCOLATE)
+            .build();
+}
```
Comment (ScorpiusSigma) [inline src/test/java/seedu/wife/testutil/TypicalTag.java:19]: can we have a getTypicalTag method and add 2 more static tags for NEW and USED

**Review by ScorpiusSigma — CHANGES_REQUESTED** [review]

File: src/test/java/seedu/wife/testutil/TypicalFood.java (hunk 1 above)
Comment (Jnjy) [reply src/test/java/seedu/wife/testutil/TypicalFood.java:59]: May I clarify that by this do you mean that there are combination of item with tags and without tags?

i.e,  `[MEIJI_WITH_TAG, CHOCOLATE_WITHOUT_TAG]`?

**Review by ScorpiusSigma — APPROVED** [review]
LGTM

**Pull request merged**